docs/use/gateway.md

```diff
@@ -57,6 +62,10 @@ Server, port, and secure_flag are only required if changing connection to anothe
 If the new connection fails the gateway will fallback to the previous connection.
 :::
 
+Define 'MQTTsetMQTT' to activate this
+```
+build_flags = '-DMQTTsetMQTT'
+```
```

Comment:
Rather than defining it as a add, I would prefer to make it default for ESP and remove it for the RFBridge:

```ini
[com-esp]
lib_deps =
  ${env.lib_deps}
  ${libraries.wifimanager}
build_flags =
  ${env.build_flags}
  '-DsimpleReceiving=true'
  '-DZmqttDiscovery="HADiscovery"'
  '-DTRACE=1'
  '-DCONFIG_BT_NIMBLE_ROLE_PERIPHERAL_DISABLED'
  '-DCONFIG_BT_NIMBLE_ROLE_BROADCASTER_DISABLED'
  '-DMQTTsetMQTT'
  ;'-DCORE_DEBUG_LEVEL=4'
```

```ini
[env:rfbridge]
platform = ${com.esp8266_platform}
board = esp8285
lib_deps =
  ${com-esp.lib_deps}
  ${libraries.esp8266_mdns}
build_flags =
  ${com-esp.build_flags}
  '-DZgatewaySRFB="SRFB"'
  '-DLED_INFO=13'
  '-DLED_INFO_ON=0'
  '-DGateway_Name="OpenMQTTGateway_SRFB"'
  '-UMQTTsetMQTT' ;We remove this function to have sufficient FLASH available for OTA, you should also use ESPWifiManualSetup to save flash memory and have OTA working
board_build.flash_mode = dout
```